github/orgs.go

```diff
@@ -213,3 +219,25 @@ func (s *OrganizationsService) Edit(ctx context.Context, name string, org *Organ
 
 	return o, resp, nil
 }
+
+// List installations for an organization.
+//
+// GitHub API docs: https://developer.github.com/v3/orgs/#list-installations-for-an-organization
+func (s *OrganizationsService) ListInstallations(ctx context.Context, org string) (*OrganizationInstallations, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/installations", org)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeIntegrationPreview)
```

Comment:
Looking at the [API docs](https://developer.github.com/v3/orgs/#list-installations-for-an-organization), this should be `mediaTypeMemberAllowedRepoCreationTypePreview`.  I'm not sure why they tied it to that set of preview APIs.  /shrug

Reply:
oops, I was looking at the wrong docs.  You have the right one here :)